### ISSUE TITLE
When user sends an attachment, the attachment is not reviewable in transcript via keyboard navigation with AT (related to #3814)

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3835]
## Version
4.13.0

## Describe the bug

If a user sends an attachment, and then reviews the transcript, the content of the attachment (filename and size) is not reviewable

## Steps to reproduce

1. Go to MockBot/playground
1. Send an attachment to Web Chat (empty txt file is fine)
1. Notice that AT reads the content of the attachment (expected)
1. Navigate to the attachment by keyboard
2. Notice that AT says 'You said, 1 attachment', but doesn't not read filename or size.

## Expected behavior

Filename and size should be read by AT when focused on that attachment in the transcript.

[Bug]
